File: python_works/oops1.py
# class Student:
#     name="Chottu"
#     roll_no=103
#     course="BCA"
#     def fun(self):
#         print("Hello")
# ob=Student()
# print(ob)
# print(ob.name)
# print(ob.roll_no)
# print(ob.course)
# ob.fun()



class Student:                                        #default constructor
    def __init__(self):
        print("Object created")
    name="Chottu"
    roll_no=103
    course="BCA"
    def fun(self):
        print("Hello")

ob=Student()

oc=Student()

File: js_works/oops1.js
// class Student{
//     constructor(){
//         name="Ajith"
//         age=20
//         display(){
//             console.log("Hello")
//         }
//     }
// }
// s1=new Student()
// console.log(s1.name)
// console.log(s1.age)
// s1.display()


// class Student{
//     constructor(){
//         this.name=n
//         this.age=a
//     }
//     display(){
//         console.log('name:$(this.name),age:$(this.age)')
//     }
// }
// s1=new Student()
// s1.display()






class School{
    sc_name="ppptttmmm"
}
class Student extends School{
    name="Ajith"
}
s1=new Student()
console.log(s1.sc_name)



baleno={
    manufacturer:"Suzuki",
    fuel:"petrol",
    color:"black",
    transmission:["manual","amt"]
}

glanza={
    manufacturer:"Toyota",
}

glanza.__proto__=baleno
console.log(glanza.color)
console.log(glanza.fuel)
console.log(glanza.transmission)
console.log(glanza.manufacturer)